src/omnicore/rpc.cpp

```diff
@@ -53,6 +55,10 @@ using std::vector;
 using namespace json_spirit;
 using namespace mastercore;
 
+bool CompareRPCSortKey(const std::pair<std::string, Object>& firstJSONObj, const std::pair<std::string, Object>& secondJSONObj)
```

Comment:
This seems to be unused now.

As a side note, but this is probably not useful: CMPTransaction objects are sortable:
```c++
bool operator<(const CMPTransaction& other) const
{
    if (block != other.block) return block > other.block;
    return tx_idx > other.tx_idx;
}
```

And it should work (untested) to use `std::sort` on a vector of `CMPTransaction` or put them into a `std::set`.